File: Iniciante/1134.py
alcool = 0
gasolina = 0
diesel = 0

while True:
  valor = int(input())

  if (valor == 1):
    alcool += 1
  elif (valor == 2):
    gasolina += 1
  elif (valor == 3):
    diesel += 1
  elif (valor == 4):
    break

print("MUITO OBRIGADO")
print("Alcool: {}".format(alcool))
print("Gasolina: {}".format(gasolina))
print("Diesel: {}".format(diesel))

File: Iniciante/1134.js
let valor, alcool = 0, gasolina = 0, diesel = 0;

for (let i = 0; i < lines.length; i++) {
  valor = Number(lines[i]);
  if (valor == 1) {
    alcool++;
  } else if (valor == 2) {
    gasolina++;
  } else if (valor == 3) {
    diesel++;
  } else if (valor == 4) {
    break;
  }
}

console.log(`MUITO OBRIGADO
Alcool: ${alcool}
Gasolina: ${gasolina}
Diesel: ${diesel}`);
